### DISCUSSION TITLE
shouldUseVirtualFocus missing in AriaListBoxOptions?

### DISCUSSION THREAD

**Discussion by stefanprobst** [adobe/react-spectrum#1624] [Q&A] [answered]
The [`useComboBox` docs example](https://react-spectrum.adobe.com/react-aria/useComboBox.html#example) passes `shouldUseVirtualFocus` to `useListBox`, however it looks like it's missing from the `AriaListBoxOptions` type. Is this a mistake in the docs or in the type?

```ts
useListBox({ shouldUseVirtualFocus: true }, state, ref)
// Object literal may only specify known properties, and 'shouldUseVirtualFocus' does not exist in type 'AriaListBoxOptions<T>'
```

**Comment by LFDanLu** [comment, marked as answer]
@stefanprobst Looks like this is a mistake in the type, `useListBox` needs `shouldVirtualFocus` passed to it so that the ListBox itself doesn't become tabbable.
